src/main/java/org/jitsi/videobridge/EndpointConnectionStatus.java

```diff
@@ -112,12 +112,12 @@ public void run()
             logger.error("Endpoint connection monitoring is already running");
         }
 
-        if (FirstTransferTimeoutProperty.getValue() <= MaxInactivityLimitProperty.getValue())
+        if (Config.getFirstTransferTimeout() <= Config.getMaxInactivityLimt())
         {
             throw new IllegalArgumentException(
                 String.format("FIRST_TRANSFER_TIMEOUT(%s) must be greater"
```

Comment:
Should this exception message reference the new spelling of the property names?

Reply:
yeah, i think i'll tweak it to reflect the property instead of the property name.  makes me think it'd also be cool if we could express this in the definitions themselves, maybe...something to think about for the future.